pkg/features/list.go

```diff
@@ -44,4 +44,6 @@ var (
 
 	// CentralEvents enables APIs (including collection) and UI pages for Central events.
 	CentralEvents = registerFeature("Enable Central events", "ROX_CENTRAL_EVENTS", false)
+
+	ScannerV4 = registerFeature("Enable Scanner v4", "ROX_SCANNER_V4", true)
```

Comment:
This should be replaced with the env var we made instead. That being said, I am not quite sure how to access normal env vars in Helm (currently doing .FeatureFlags.ROX_SCANNER_V4), so that's something to figure out